packages-exp/installations-exp/src/api/get-id.ts

```diff
@@ -23,6 +23,7 @@ import { FirebaseInstallations } from '../interfaces/public-types';
 /**
  * Creates a Firebase Installation if there isn't one for the app and
  * returns the Installation ID.
+ * @param installations - The Installations instance
```

Comment:
Another global nit, and I hate to occupy an engineer's time with this, but -- these and all other non-title strings should end in periods.

See the Firestore tables.  We also added init caps and periods in the FCM strings.  If possible, we should stay consistent with that.  Thanks!